addons/stock_account/models/stock.py

```diff
@@ -150,20 +177,29 @@ def replay(self):
 
     @api.multi
     def action_done(self):
+        """
+            For incoming moves, it will put the value corresponding with that move on the stock move, 
+            except for standard cost price, where it will put the standard price.  
+            For FIFO, it will also check if there was negative stock and compensate the corresponding out moves.   
+            For outgoing moves, average price, will apply the average price formula, FIFO will search the 
+            corresponding incoming moves where it can reduce the remaining_qty while standard takes the cost from the product
+        """
         qty_available = {}
         for move in self:
-            #Should write move.price_unit here maybe, certainly on incoming
+            #Should write move.price_unit here maybe, certainly on incoming moves
             if move.product_id.cost_method == 'average':
```

Comment:
why this change: average price should look for every product available in the company, even if they're in transit between two warehouses